### ISSUE TITLE
Missing images on blogroll

### ISSUE THREAD

**Issue by gursuj** [maxboeck/mxb#87]
Hello, I stumbled across your website and saw that some images on the blogroll page were missing. The alt text also overlaps with the description. Didn't find any other way to contact you :p 

<img width="414" height="290" alt="Image" src="https://github.com/user-attachments/assets/a388756d-7db7-4ba0-be51-c76767d5af75" />
